router/transformer/transformer.go

```diff
@@ -32,19 +34,37 @@ const (
 
 // HandleT is the handle for this class
 type HandleT struct {
-	tr                                 *http.Transport
-	client                             *http.Client
+	tr *http.Transport
+	// http client for router transformation request
+	client *http.Client
+	// Mockable http.client for transformer proxy request
+	tfProxyClient sysUtils.HTTPClientI
+	// http client timeout for transformer proxy request
+	tfProxyTimeout                     time.Duration
 	transformRequestTimerStat          stats.RudderStats
 	transformerNetworkRequestTimerStat stats.RudderStats
 	transformerProxyRequestTime        stats.RudderStats
 	logger                             logger.LoggerI
 }
 
+type ProxyRequestParams struct {
+	ResponseData integrations.PostParametersT
+	DestName     string
+	JobId        int64
+	BaseUrl      string
+}
+
+type HTTPProxyResponse struct {
```

Comment:
this is an internal structure, no need for it to be exported
```suggestion
type httpProxyResponse struct {
```

Reply:
Done